### ISSUE TITLE
Error after execute fish_update_completions 

### ISSUE THREAD

**Issue by kevariable** [fish-shell/fish-shell#7386]
fish, version 3.1.2
Linux kevariable 5.8.12-arch1-1 #1 SMP PREEMPT Sat, 26 Sep 2020 21:42:58 +0000 x86_64 GNU/Linux

after execute `fish_update_completions`

i got this error

```python
Traceback (most recent call last):
  File "/usr/share/fish/tools/create_manpage_completions.py", line 1066, in <module>
    file_paths.extend(get_paths_from_man_locations())
  File "/usr/share/fish/tools/create_manpage_completions.py", line 965, in get_paths_from_man_locations
    manpath, err_data = proc.communicate()
AttributeError: 'NoneType' object has no attribute 'communicate'
```

**Comment by faho** [comment]
This happens when you have neither $MANPATH set, nor a program called `manpath` or a man that understands `--path` in $PATH. Given that arch's man from man-db understands `--path` your system is at least pretty unusual, if not broken outright.

Anyway, this is fixed by 02e9486559d0a49568691b7a3c1daac12d240275, we no longer call things this way.

**Comment by kevariable** [comment]
i have done with installed man :) thanks
